ci/vars-nightly_bundles.yml

```diff
@@ -0,0 +1,7 @@
+---
+# from: https://github.com/infrawatch/service-telemetry-operator/pull/437
+# ansible-playbook -e __service_telemetry_storage_ephemeral_enabled=true -e __local_build_enabled=false -e __deploy_from_bundles_enabled=true -e __service_telemetry_bundle_image_path=quay.io/infrawatch-operators/service-telemetry-operator-bundle:nightly-head -e __smart_gateway_bundle_image_path=quay.io/infrawatch-operators/smart-gateway-operator-bundle:nightly-head --skip-tags bundle_registry_tls_ca --skip-tags bundle_registry_auth build/run-ci.yaml
+
+__local_build_enabled: false
+__deploy_from_bundles_enabled: true
+__service_telemetry_storage_ephemeral_enabled: true
```

Comment:
Does CRC provide pre-configured PVs? I'd actually prefer if we could stop using ephemeral storage in our testing scenarios if possible. It's probably not a huge deal, but since our documentation uses persistent storage, it'd be ideal to test that part of the code path.

Reply:
Note that ephemeral storage was only used historically because there is no PV in quicklab, where our tests were originally running. We should be trying to use persistent storage whenever possible.

Reply:
Agree, if CRC will provide PVs, that would be ideal.

Reply:
That's something to try later, I think. This set of jobs (local_build, local_build-index_deploy, nightly-bundles) need to get merged first so we have testing on PRs

Reply:
I've added a spike to the next sprint to try answer these questions, and scope adding PVs. It might be trivial, not possible or require changes in rdo-jobs